consensus/core/src/commit_syncer.rs

```diff
@@ -636,7 +703,7 @@ impl<C: NetworkClient> Inner<C> {
     }
 }
 
-struct FetchState {
+struct PeerState {
```

Comment:
`PeerFetchState` or `PeerFetchStatus`?

Reply:
I think the fields are unrelated to the status of fetches. In future this will be replaced by a generic "peer selector".